src/Microsoft.ML.Recommender/RecommenderCatalog.cs

```diff
@@ -33,6 +33,7 @@ public sealed class RecommendationCatalog : TrainCatalogBase
         internal RecommendationCatalog(IHostEnvironment env)
             : base(env, nameof(RecommendationCatalog))
         {
+            Host = env;
```

Comment:
>Host [](start = 12, length = 4)

You're passing this in to the base constructor. Why are you setting it here?

Reply:
removed

---
In reply to: [256217269](https://github.com/dotnet/machinelearning/pull/2524#discussion_r256217269) [](ancestors = 256217269)